src/Main/Main.cc

```diff
@@ -79,13 +78,8 @@ int main(int argc, char* argv[]) {
         casacore::LogSink::globalSink(carta_log_sink);
         casacore::LogIO casacore_log(log_sink);
 
-        if (settings.wait_time >= 0) {
-            Session::SetExitTimeout(settings.wait_time);
-        }
-
-        if (settings.init_wait_time >= 0) {
-            Session::SetInitExitTimeout(settings.init_wait_time);
-        }
+        Session::SetExitTimeout();
+        Session::SetInitExitTimeout();
```

Comment:
In these two specific cases, the previous behaviour was cleaner and should be restored.